ui/src/app/images/components/ImagesTable/ImagesTable.tsx

```diff
@@ -3,14 +3,10 @@ import { useFormikContext } from "formik";
 
 import DoubleRow from "app/base/components/DoubleRow";
 import type { ImageValue } from "app/images/types";
-import type {
-  BootResource,
-  BootResourceUbuntuRelease,
-} from "app/store/bootresource/types";
+import type { BootResource } from "app/store/bootresource/types";
 import { splitResourceName } from "app/store/bootresource/utils";
 
 type Props = {
-  releases: BootResourceUbuntuRelease[];
```

Comment:
I've removed this prop so the component can be used for image types beside Ubuntu images